common/scripts/lib/_parseArgs.sh

```diff
@@ -339,14 +339,15 @@ __print_help() {
     help            Print this message
     clean           Remove shippable containers and configurations
     info            Print information about current installation
+    list            Print available admiral versions
   "
   exit 0
 }
 
 __print_list() {
   printf "\nAvailable Admiral versions are:\n"
   git tag
-  printf "\nTo upgrade to a version use: $./admiral.sh upgrade <version>\n\n"
+  printf "\nTo upgrade to a version use: #./admiral.sh upgrade <version>\n\n"
```

Comment:
lets keep it `$`. we're not using `#` anywhere so if we make this change, we'll have to do it globally